source/adapters/cuda/device.cpp

```diff
@@ -1078,7 +1078,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(ur_device_handle_t hDevice,
   case UR_DEVICE_INFO_COMPONENT_DEVICES:
   case UR_DEVICE_INFO_COMPOSITE_DEVICE:
     // These two are exclusive of L0.
-    return ReturnValue(0);
+    return UR_RESULT_ERROR_UNSUPPORTED_FEATURE;
```

Comment:
can you make the return `UR_RESULT_ERROR_UNSUPPORTED_ENUMERATION` instead? that's what we have in the spec for unsupported queries and it's what the cts expects if a query is allowed to be unsupported

Reply:
Sure, we can do that. It looks a little weird in the SYCL runtime because it currently maps to `PI_ERROR_INVALID_VALUE`, but I suppose "invalid" means "unsupported" when it comes to info queries anyway.

Reply:
yeah that's a bad mapping, could change it to map to `PI_ERROR_UNSUPPORTED_FEATURE` in the corresponding llvm pr or just leave it and wait for the UR port to sort it out

Reply:
I figure in the vast majority of cases, "invalid" from an info query means the query is either not recognized or intentionally not supported. In both those cases, the feature isn't supported, so it's probably fine. 😄